test/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ list( APPEND test_input
   testinput/b001xx007.20200310.bufr
   testinput/viirs_aod.nc
   testinput/tropomi_no2.nc
+# testinput/gdas.t12z.1bmhs.tm00.bufr_d
```

Comment:
Don't forget about adding the YAML file to this list too.

Reply:
oops, thanks!